onnx/defs/nn/defs.cc

```diff
@@ -833,7 +847,7 @@ computes the output.)DOC";
         OPTIONAL_VALUE);
     schema.Attr(
         "auto_pad",
-        auto_pad_doc,
+        conv_auto_pad_14_doc,
```

Comment:
If behavior of Conv is changing then the behavior for corresponding quantization ops should change too